test/clojure/core_test/uuid_qmark.cljc

```diff
@@ -0,0 +1,39 @@
+(ns clojure.core-test.uuid-qmark
+  (:require [clojure.test :refer [are deftest testing]]
+            [clojure.core-test.portability #?(:cljs :refer-macros :default :refer) [when-var-exists]]))
+
+(when-var-exists uuid?
+  (deftest test-uuid?
+
+    (testing "uuids"
+      (are [x] (uuid? x)
+               #uuid "f81d4fae-7dec-11d0-a765-00a0c91e6bf6"
+               (parse-uuid "f81d4fae-7dec-11d0-a765-00a0c91e6bf6")
+               (random-uuid)))
```

Comment:
Might be a good idea to test the "nil" uuid (i.e. `00000000-0000-0000-0000-000000000000`) occasionally runtimes treat this UUID as a special case, which would be good to catch

Reply:
Thanks, added case for nil uuid.